### PR TITLE
Navbar and crafted sections

### PR DESCRIPTION
**Pull request by jacopobacci**

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+a {
+  text-decoration: none;
+}
+
+* {
+  margin: 0;
+  padding: 0;
```
Comment (glgoose) [inline assets/css/styles.css:7]: would be good to also add `box-sizing: border-box`

File: index.html [hunk 2]
```diff
@@ -5,22 +5,37 @@
...
+      </nav>
+      <section class="crafted">
+        <span class="crafted__left-arrow">&LeftArrow;</span>
+        <h1>Crafted for Business</h1>
+        <p>We blend insights and strategy to create digital products for forward-thinking organisations.</p>
+        <div class="crafted__buttons">
+          <a href="#"><button>Get Started</button></a>
+          <a href="#"><button>Download</button></a>
+        </div>
+        <img src="/assets/img/slider/2.png" alt="" />
```
Comment (glgoose) [inline index.html]: I guess for a slider img like this alt text is indeed not needed

**Review by glgoose — CHANGES_REQUESTED** [review]
you pushed the `.DS_Store` files, can you remove them and add a `.gitignore` which ignores those DS files

otherwise looks good, one other small change

**Review by glgoose — CHANGES_REQUESTED** [review]
could you also delete `.DS_Store` & `assets/.DS_Store`?

It may be that I originally added them to the repo and they got changed because of different settings on your Mac. We can just delete them now and with the gitignore in place won't be added anymore in the future

**Review by glgoose — CHANGES_REQUESTED** [review]
`assets/.DS_Store` is still present, would need to manually remove that as well

**Comment by jacopobacci** [comment]
removed!

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
+**/.DS_Store
```
Comment (glgoose) [inline .gitignore]: `*.DS_Store` should be enough
I just found this handy overview: https://www.atlassian.com/git/tutorials/saving-changes/gitignore

**Review by glgoose — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (jacopobacci) [reply .gitignore]: ok good to know. i updated again

**Review by glgoose — APPROVED** [review]

**Pull request merged**